pkg/roachprod/vm/azure/azure.go

```diff
@@ -854,6 +854,77 @@ func (p *Provider) createNIC(
 	return
 }
 
+// securityRules returns an array of TCP security rules and contains
+// a list of well-known, and roachtest specific ports.
+func securityRules() *[]network.SecurityRule {
+	allowTCP := func(name string, priority int32, direction network.SecurityRuleDirection, destPortRange string) network.SecurityRule {
+		suffix := ""
+		switch direction {
+		case network.SecurityRuleDirectionInbound:
+			suffix = "_Inbound"
+		case network.SecurityRuleDirectionOutbound:
+			suffix = "_Outbound"
+		default:
+		}
+		res := network.SecurityRule{
+			Name: to.StringPtr(name + suffix),
+			SecurityRulePropertiesFormat: &network.SecurityRulePropertiesFormat{
+				Priority:                 to.Int32Ptr(priority),
+				Protocol:                 network.SecurityRuleProtocolTCP,
+				Access:                   network.SecurityRuleAccessAllow,
+				Direction:                direction,
+				SourceAddressPrefix:      to.StringPtr("*"),
+				SourcePortRange:          to.StringPtr("*"),
+				DestinationAddressPrefix: to.StringPtr("*"),
+				DestinationPortRange:     to.StringPtr(destPortRange),
+			},
+		}
+		return res
+	}
+
+	namedInbound := map[string]string{
+		"SSH":            "22",
+		"HTTP":           "80",
+		"HTTPS":          "43",
+		"CockroachPG":    "26257",
+		"CockroachAdmin": "26258",
+		"Grafana":        "3000",
+		"Prometheus":     "9090",
+		"Kafka":          "9092",
+	}
+
+	// The names for these are generated in the form Roachtest_<index>_Inbound.
+	// The mapped roachtests are not exhaustive, and at some point will be
+	// cumbersome to keep adding exceptions for. We may need to consider removing
+	// all rules.
+	genericInbound := []string{
```

Comment:
`workload` has another range for prom metrics: `2112-2120` and `33333` for `pprof` :) (It's in `pkg/workload/cli/run.go`.)

Reply:
Thanks - updated!